oar/core/config_store.py

```diff
@@ -185,3 +185,9 @@ def get_slack_app_token(self):
         Get slack app token
         """
         return os.environ.get(ENV_VAR_SLACK_APP_TOKEN)
+
+    def get_google_app_passwd(self):
+        """
+        Get google account application password
+        """
+        return os.environ.get(ENV_APP_PASSWD)
```

Comment:
@wewang58 I add enhancement for getting env var in https://github.com/openshift/release-tests/pull/8, you can rebase and invoke `_get_env_var(self, var)` to get env var, if the var is not exported, exception will be raised.